simulation/navigator_gazebo/models/wamv/wamv.sdf

```diff
@@ -36,69 +36,78 @@
-        <distortionK2>0.0</distortionK2>
-        <distortionK3>0.0</distortionK3>
-        <distortionT1>0.0</distortionT1>
-        <distortionT2>0.0</distortionT2>
-      </plugin>
+
+      <sensor name="front_left_camera" type="camera">
+        <camera>
+          <pose> 0.016378 0.1 0.198208 8.83556166648e-07 0.257608854236 0.0230515819239 </pose>
+          <horizontal_fov>1.047</horizontal_fov>
+          <image>
+            <width>960</width>
+            <height>600</height>
+          </image>
+          <distortion>
+            <k1>0</k1>
+            <k2>0</k2>
+            <k3>0</k3>
+            <p1>0</p1>
+            <p2>0</p2>
+          </distortion>
+        </camera>
+        <plugin name="stereo_left_camera_controller" filename="libgazebo_ros_camera.so">
+          <alwaysOn>true</alwaysOn>
+          <updateRate>30</updateRate>
+          <cameraName>stereo/left</cameraName>
+          <imageTopicName>image_raw</imageTopicName>
+          <cameraInfoTopicName>camera_info</cameraInfoTopicName>
+          <frameName>stereo_left_cam</frameName>
+          <hackBaseline>0</hackBaseline>
+          <distortionK1>0</distortionK1>
+          <distortionK2>0</distortionK2>
+          <distortionK3>0</distortionK3>
+          <distortionT1>0</distortionT1>
+          <distortionT2>0</distortionT2>
+        </plugin>
+      </sensor>
+
+      <sensor name="front_right_camera" type="camera">
+        <camera>
+          <pose> 0.016378 0.08 0.198208 -6.01198519164e-06 0.242958281975 0.0188484790053 </pose>
+          <horizontal_fov>1.047</horizontal_fov>
+          <image>
+            <width>960</width>
+            <height>600</height>
+          </image>
+          <distortion>
+            <k1>0</k1>
+            <k2>0</k2>
+            <k3>0</k3>
+            <p1>0</p1>
+            <p2>0</p2>
+          </distortion>
+        </camera>
+        <plugin name="stereo_right_camera_controller" filename="libgazebo_ros_camera.so">
+          <alwaysOn>true</alwaysOn>
+          <updateRate>30</updateRate>
+          <cameraName>stereo/right</cameraName>
+          <imageTopicName>image_raw</imageTopicName>
+          <cameraInfoTopicName>camera_info</cameraInfoTopicName>
+          <frameName>stereo_right_cam</frameName>
+          <hackBaseline>0</hackBaseline>
+          <distortionK1>0</distortionK1>
+          <distortionK2>0</distortionK2>
+          <distortionK3>0</distortionK3>
+          <distortionT1>0</distortionT1>
+          <distortionT2>0</distortionT2>
+        </plugin>
       </sensor>
 
       <sensor name="right_camera" type="camera">
         <camera>
-          <pose>-0.2032 -0.4445 0.24765 0 0 -1.5709</pose>
+          <pose> -0.142622 -0.516 0.195208 -5.92339209527e-06 -0.0816781931371 -1.64410186098 </pose>
           <horizontal_fov>1.047</horizontal_fov>
           <image>
-            <width>644</width>
-            <height>482</height>
+            <width>960</width>
+            <height>600</height>
```

Comment:
I set up the down camera based on the right camera, so apply any resolution changes to it as well.